### PR TITLE
#230 add xslt update step into ubuntu worflow file

### PR DESCRIPTION
**Pull request by CAMOBAP**
- #230

### CODE REVIEW

**Comment by CAMOBAP** [comment]
Tested https://github.com/metanorma/mn-native-pdf/runs/1847900383?check_suite_focus=true#step:11:1 it works, but questions in https://github.com/metanorma/mn-native-pdf/issues/230#issuecomment-774606541 still need to be answered

File: Makefile [hunk 1]
```diff
@@ -324,4 +324,30 @@ else
...
+	$(MAKE) update-xslts-in-processor MN_PROCR=bipm
+	$(MAKE) update-xslts-in-processor MN_PROCR=csa
+	$(MAKE) update-xslts-in-processor MN_PROCR=cc XSLT_PREFIX=csd
+	$(MAKE) update-xslts-in-processor MN_PROCR=gb
+	$(MAKE) update-xslts-in-processor MN_PROCR=iec
+	$(MAKE) update-xslts-in-processor MN_PROCR=iho
+	$(MAKE) update-xslts-in-processor MN_PROCR=iso
+	$(MAKE) update-xslts-in-processor MN_PROCR=itu
+	$(MAKE) update-xslts-in-processor MN_PROCR=iec
+	$(MAKE) update-xslts-in-processor MN_PROCR=m3aawg XSLT_PREFIX="{m3d,jcgm}"
```
Comment (opoudjis) [inline Makefile:347]: jcgm is a variant of bipm, not m3aawg

**Review by opoudjis — APPROVED** [review]

**Comment by CAMOBAP** [comment]
@Intelligent2013 looks like the symlink issue can be fixed by this https://stackoverflow.com/questions/18411200/git-unable-to-create-symlink-file-name-too-long

**Comment by Intelligent2013** [comment]
> 
> 
> @Intelligent2013 looks like the symlink issue can be fixed by this https://stackoverflow.com/questions/18411200/git-unable-to-create-symlink-file-name-too-long

@CAMOBAP github converted normal file with content into symlink... I don't know a reason, but now it fixed.

File: Makefile (hunk 1 above)
Comment (CAMOBAP) [reply Makefile]: Fixed

**Comment by CAMOBAP** [comment]
All processors related PRs are merged, I have renamed xslts, waiting for CI

**Comment by ronaldtse** [comment]
Thanks @CAMOBAP !

**Comment by CAMOBAP** [comment]
@Intelligent2013 may I kindly ask you to review the PR again because I added some more changes on top

**Review by Intelligent2013 — APPROVED** [review]

**Pull request merged**